docs/griptape-framework/misc/src/events_3.py

```diff
@@ -16,13 +15,11 @@
     ]
 )
 
-pipeline = Pipeline(
-    config=OpenAiStructureConfig(prompt_driver=OpenAiChatPromptDriver(model="gpt-4o", stream=True)),
-)
-
+pipeline = Pipeline()
 pipeline.add_tasks(
     ToolkitTask(
         "Based on https://griptape.ai, tell me what griptape is.",
+        prompt_driver=OpenAiChatPromptDriver(model="gpt-4o", stream=True),
```

Comment:
I suppose passing `stream` in to the driver works, but I think its a little confusing still. I'd expect either  `pipeline.run(stream=True)` or maybe `Pipeline(stream=True)`...

Reply:
A Pipeline may not contain any Tasks that use Prompt Drivers therefore streaming would be irrelevant. For Agents, it's always either be a Prompt Task or Toolkit Task (unless a user overrides `tasks`).